### PR TITLE
Remove JDBI dependency to help with build-to-native

### PR DESCRIPTION
**Pull request by EricWittmann**

### CODE REVIEW

File: app/src/main/java/io/apicurio/registry/storage/impl/sql/mappers/RuleConfigurationDtoMapper.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    public static final RuleConfigurationDtoMapper instance = new RuleConfigurationDtoMapper();
+
+    /**
+     * Constructor.
+     */
+    private RuleConfigurationDtoMapper() {
+    }
+
+    /**
+     * @see org.jdbi.v3.core.mapper.RowMapper#map(java.sql.ResultSet, org.jdbi.v3.core.statement.StatementContext)
```
Comment (carlesarnal) [inline app/src/main/java/io/apicurio/registry/storage/impl/sql/mappers/RuleConfigurationDtoMapper.java:39]: I've missed this in my previous review, I don't think it's related to the compilation issues but we should remove the reference to jdbi.

Comment (EricWittmann) [reply app/src/main/java/io/apicurio/registry/storage/impl/sql/mappers/RuleConfigurationDtoMapper.java]: I've updated these, thanks.

**Comment by EricWittmann** [comment]
OK I finally have this updated and merged with latest on `master`.  Verify and k8s tests are green.  @famartinrh can you have a look at the integration test failures?

**Comment by EricWittmann** [comment]
All green! Thanks @famartinrh 

@Apicurio/developers any objections to this change?

**Comment by famartinrh** [comment]
just one, move all the new code to a separated package? because this is almost a new library it makes sense to have it separated

**Pull request merged**